qa/suites/upgrade/client-upgrade-kraken/kraken-client-x/rbd/1-install/kraken-client-x.yaml

```diff
@@ -4,7 +4,7 @@ tasks:
     exclude_packages: ['ceph-mgr','libcephfs2','libcephfs-devel','libcephfs-dev']
 - print: "**** done install jewel"
```

Comment:
should we do s/jewel/kraken/ here?

Reply:
actually it depends if we are running this test against jewel, it's jewel. if we are running this test against luminous, it'd be luminous.

Reply:
even when the install block here says `branch: kraken`?

Reply:
oh, that's right.